src/main/java/co/omise/models/Source.java

```diff
@@ -113,44 +121,59 @@ public void setTerminalId(String terminalId) {
         @JsonProperty("terminal_id")
         private String terminalId;
 
-        public Create amount(long amount) {
+        @Override
+        protected String method() {
+            return POST;
+        }
+
+        @Override
+        protected HttpUrl path() {
+            return buildUrl(Endpoint.API, "sources");
+        }
+
+        @Override
+        protected RequestBody payload() throws IOException {
+            return serialize();
+        }
+
+        public CreateRequestBuilder amount(long amount) {
             this.amount = amount;
             return this;
         }
 
-        public Create currency(String currency) {
+        public CreateRequestBuilder currency(String currency) {
             this.currency = currency;
             return this;
         }
 
-        public Create type(SourceType type) {
+        public CreateRequestBuilder type(SourceType type) {
```

Comment:
Should we add support for other types in this PR?

Reply:
What are the other source types?

Reply:
@Namiii Do you will do the installments in this PR or a new PR?

Reply:
@pitiphong-p if everything else is good, shall we close this PR and I'll start working on the `installments` PR.